websocketpp/frame.hpp

```diff
@@ -234,17 +234,17 @@ struct basic_header {
 /// The variable size component of a WebSocket frame header
 struct extended_header {
     extended_header() {
-        std::fill_n(this->bytes,MAX_EXTENDED_HEADER_LENGTH,0x00);
+        std::fill_n(this->bytes,MAX_EXTENDED_HEADER_LENGTH,static_cast<uint8_t>(0x00));
```

Comment:
I think value initialization (as in https://github.com/zaphoyd/websocketpp/pull/783 ) is a better workaround for these.